openapi3gen/openapi3gen.go

```diff
@@ -47,6 +58,12 @@ func UseAllExportedFields() Option {
 	return func(x *generatorOpt) { x.useAllExportedFields = true }
 }
 
+func CreateTypeNameGenerator(tngnrt TypeNameGenerator) Option {
+	return func(x *generatorOpt) {
+		x.typeNameGenerator = tngnrt
+	}
```

Comment:
nit: please single-line this

Reply:
@EnriqueL8 comment fixed. There is a PR into your repo